### DISCUSSION TITLE
G3-4 Water Flow sensor | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1515] [Q&A]
# G3-4 Water Flow sensor | Seeed Studio Wiki

Water flow sensor consists of a plastic valve body, a water rotor, and a hall-effect sensor. When water flows through the rotor, rotor rolls. Its speed changes with different rate of flow. The hall-effect sensor outputs the corresponding pulse Signal.

https://wiki.seeedstudio.com/G3-4_Water_Flow_sensor/

**Comment by xhaniee** [comment]
I'm studying a formula that states: "Pulse frequency (Hz) in Horizontal Test = 5.5Q, where Q is the flow rate in L/min. (Results in the +/- 3% range)."

I'm curious to understand the origin of the factor 5.5 in this formula. Does anyone know where this number comes from? Is it derived from a specific experiment or based on existing theory?

Additionally, how is the +/- 3% range determined for the results? Is this obtained from experimental testing, or is there a theoretical basis for it?
